x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_index/pipelines/ml_inference/text_expansion_callout_logic.ts

```diff
@@ -48,6 +48,11 @@ interface TextExpansionCalloutActions {
   textExpansionModel: FetchTextExpansionModelApiLogicActions['apiSuccess'];
 }
 
+export interface TextExpansionCalloutErrors {
```

Comment:
Nit: since this contains a single error, I'd rename it to `TextExpansionCalloutError`.
```suggestion
export interface TextExpansionCalloutError {
```

Reply:
Updated